src/FieldEncryptProcessEntities.php

```diff
@@ -338,4 +338,55 @@ public function updateStoredField($field_name, $field_entity_type, $original_enc
     }
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function entitySetCacheTags(ContentEntityInterface $entity, &$build) {
+    $cache_exclude_fields = $this->getCacheExcludeFields($entity);
+    foreach ($cache_exclude_fields as $field_name) {
+      $build[$field_name]['#cache']['max-age'] = 0;
```

Comment:
IOW: this should not even be necessary, given changes elsewhere.

Furthermore, the function name doesn't match what it does.